docs/guide/assets.md

```diff
@@ -35,6 +35,15 @@ module.exports = {
 
 Sometimes you may need to provide static assets that are not directly referenced in any of your markdown or theme components - for example, favicons and PWA icons. In such cases you can put them inside `.vuepress/public` and they will be copied to the root of the generated directory.
 
+When calling assets from the `.vuepress/public` directory into non-markdown files, use absolute paths:
```

Comment:
There should be no distinction between file types. To request any assets under the `public` dir, you should ONLY use absolute paths instead of relative paths. 

Otherwise, the file will not only be copied, but will also be built.

Reply:
I've adjusted the sentence to remove file-type distinction and make it clear that the user should always use absolute paths.